docs(ai-agents): Update LangChain quickstart to use create_agent (#78152)
airbytehq/airbyte: docs/ai-agents/get-started/developer-quickstart/tutorial-langchain.md

### Define the agent

Create a LangChain chat model and an agent with a system prompt that describes its purpose:

```python title="agent.py"
llm = ChatOpenAI(model="gpt-4o")

agent = create_agent(
    model=llm,
    tools=[github_execute],
    system_prompt=(
        "You are a helpful assistant that can access GitHub data through the "
        "github_execute tool. Be concise and accurate."
    ),
)
```

- `ChatOpenAI(model="gpt-4o")` creates an OpenAI chat model. You can use a different model by changing the model string. For example, use `"gpt-4o-mini"` to lower costs. LangChain also supports [other providers](https://python.langchain.com/docs/integrations/chat/) like Anthropic and Google.

> - `create_agent` creates an agent that reasons about which tools to call based on the user's input. LangChain's agent runtime is built on LangGraph, but you don't need to import LangGraph directly for this quickstart.

Author: ```suggestion
- `create_agent` creates an agent that reasons about which tools to call based on your input. LangChain's agent runtime is built on LangGraph, but you don't need to import LangGraph directly for this quickstart.
```
